src/hangar/checkout.py

```diff
@@ -127,6 +134,236 @@ def __verify_checkout_alive(self):
+
+        .. seealso:
+
+            :meth:`~hangar.checkout.ReaderCheckout.get`
+
+        """
+        try:
+            tmpconman = not self._is_conman
+            if tmpconman:
+                self.__verify_checkout_alive()
+                self.__enter__()
+
+            if isinstance(index, str):
+                return self.arraysets[index]
+            elif not isinstance(index, (tuple, list)):
+                raise TypeError(f'Unknown index: {index} type: {type(index)}')
+            if len(index) > 2:
+                raise ValueError(f'index of len > 2 not allowed: {index}')
+
+            arraysets, samples = index
+            return self.get(arraysets, samples, except_missing=True)
+
+        finally:
+            if tmpconman:
+                self.__exit__()
+
+    def get(self, arraysets, samples, *, except_missing=False):
+        """View of sample data across arraysets gracefully handeling missing sample keys.
+
+        Please see :meth:`__getitem__` for full description. This method is
+        identical with a single exception: if a sample key is not present in an
+        arrayset, this method will plane a null ``None`` value in it's return
+        slot rather than throwing a ``KeyError`` like the dict style access
+        does.
+
+        Parameters
+        ----------
+        arraysets: Union[str, Iterable[str], Ellipses, slice(None)]
+
+            Name(s) of the arraysets to query. The Ellipsis operator (``...``)
+            or unbounded slice operator (``:`` <==> ``slice(None)``) can be
+            used to indicate "select all" behavior.
+
+        samples: Union[str, int, Iterable[Union[str, int]]]
+
+            Names(s) of the samples to query
+
+        except_missing: bool, **KWARG ONLY
+
+            If False, will not throw exceptions on missing sample key value.
+            Will raise KeyError if True and missing key found.
+
+        Returns
+        -------
+        Arrayset
+            single arrayset parameter, no samples specified
+
+        np.ndarray
+            Single arrayset specified, single sample key specified
+
+        List[np.ndarray]
+            Single arrayset, multiple samples array data for each sample is
+            returned in same order sample keys are recieved.
+
+        List[NamedTuple[*np.ndarray]]
+            Multiple arraysets, multiple samples. Each arrayset's name is used
+            as a field in the NamedTuple elements, each NamedTuple contains
+            arrays stored in each arrayset via a common sample key. Each sample
+            key is returned values as an individual element in the
+            List. The sample order is returned in the same order it wasw recieved.
+
+        """
+        try:
+            tmpconman = not self._is_conman
+            if tmpconman:
+                self.__verify_checkout_alive()
+                self.__enter__()
+
+            # Arrayset Parsing
+            if (arraysets is (Ellipsis, slice)):
```

Comment:
`in` instead of `is`?

Reply:
Nice catch!

Actually it's a bit of a weird case here. `Ellipses` is an object literal and needs a actual `is` check (like comparing if something is `None`), however, the comparison to `slice` needs a `isinstance` check... 

Should be `if (arraysets is Ellipsis) or isinstance(arraysets, slice):` fixing it now!